ocaml/utils/config.common.ml.in

```diff
@@ -170,15 +121,6 @@ let configuration_variables () =
   p_bool "supports_shared_libraries" supports_shared_libraries;
   p_bool "native_dynlink" native_dynlink;
   p_bool "naked_pointers" naked_pointers;
-<<<<<<< HEAD:utils/config.common.ml
-(*
-  Disabled in flambda-backend (for now)
-  p_bool "compression_supported" (Marshal.compression_supported());
-*)
```

Comment:
Shouldn't we keep that part of the diff?

Reply:
I don't think so, as this has been removed upstream.  There are CRs elsewhere in our tree where `Compression` is now used unconditionally upstream (and we don't use it).